### PR TITLE
Downloadable examples

### LINKED ISSUE #45: Add downloadable examples and tutorial code
Should also allow us to test examples and tutorials against DSL changes.

### PR DESCRIPTION
**Pull request by SamMarkowitz**
fixes #45

### CODE REVIEW

**Comment by SamMarkowitz** [comment]
Ready for circleci. I'm just not exactly sure how to make it run automatically. Tried setting up a webhook, but I guess I didn't do it right.

File: code/tutorial_code/tutorials_04/hiring/check_availability.sl [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  inputs:
+    - address
+
+  action:
+    python_script: |
+      import random
+      rand = random.randint(0, 2)
+      vacant = rand != 0
+      #print rand
```
Comment (levice14) [inline code/tutorial_code/tutorials_04/hiring/check_availability.sl:14]: I see `# instr` in several places, are those intended or just debug purposes?

Comment (SamMarkowitz) [inline code/tutorial_code/tutorials_04/hiring/check_availability.sl:14]: Intended. 

From the lesson:

"We’ve added a commented-out line, using a Python comment (#) to print the random number that was generated. We can uncomment this line during testing to see that our operation is working as expected."

**Comment by levice14** [comment]
What do we want to achive here with CircleCI? I we want to test the flows we need test files for them, like we do in the content repository - e.g. [for print](https://github.com/CloudSlang/cloud-slang-content/blob/21663770817b3e20dad4ecf7f1792d13bb63a19c/test/io/cloudslang/base/print/print_text.inputs.yaml) -- once we have the test files, I can take the build config

File: code/tutorial_code/tutorials_04/hiring/check_availability.sl (hunk 1 above)
Comment (levice14) [inline code/tutorial_code/tutorials_04/hiring/check_availability.sl:14]: :+1:

File: tutorial/circle.yml [hunk 2]
```diff
@@ -0,0 +1,108 @@
+#   (c) Copyright 2014 Hewlett-Packard Development Company, L.P.
```
Comment (levice14) [inline tutorial/circle.yml]: this config will not work in this repo (at least it is not straightforward)

Comment (SamMarkowitz) [inline tutorial/circle.yml]: Disclaimer: I don't know what I'm talking about here.

I used this file on a fork of this repository and it ran the builder and made sure everything compiled when I triggered it manually.

**Comment by SamMarkowitz** [comment]
The idea of running the builder was just to make sure everything still compiles, especially after DSL changes. It could be it's more trouble than it's worth though.

**Comment by levice14** [comment]
got it - i will check its feasabilty

**Comment by levice14** [comment]
build configured - needed to enable the project in CircleCI
- `./code` directory is compiled with `v0.9.1`
- added an icon (which should go green once merged to master)

@SamMarkowitz WDYT? can we merge this PR? please check builder [output](https://circle-artifacts.com/gh/CloudSlang/docs/3/artifacts/0/home/ubuntu/docs/cslang-builder/logs/builder.log)

**Comment by SamMarkowitz** [comment]
Yep. Looks good thanks. Merge away at your leisure.

**Pull request merged**